k8s/bases/cse-valid-idcc/kustomization.yaml

```diff
@@ -2,21 +2,9 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  - data-bridge/cse-valid-idcc-cgm-data-bridge-configmap.yaml
-  - data-bridge/cse-valid-idcc-cgm-data-bridge-deployment.yaml
-  - data-bridge/cse-valid-idcc-cgm-data-bridge-service.yaml
-  - data-bridge/cse-valid-idcc-glsk-data-bridge-configmap.yaml
-  - data-bridge/cse-valid-idcc-glsk-data-bridge-deployment.yaml
-  - data-bridge/cse-valid-idcc-glsk-data-bridge-service.yaml
-  - data-bridge/cse-valid-idcc-crac-export-corner-data-bridge-configmap.yaml
-  - data-bridge/cse-valid-idcc-crac-export-corner-data-bridge-deployment.yaml
-  - data-bridge/cse-valid-idcc-crac-export-corner-data-bridge-service.yaml
-  - data-bridge/cse-valid-idcc-crac-import-corner-data-bridge-configmap.yaml
-  - data-bridge/cse-valid-idcc-crac-import-corner-data-bridge-deployment.yaml
-  - data-bridge/cse-valid-idcc-crac-import-corner-data-bridge-service.yaml
-  - data-bridge/cse-valid-idcc-ttc-adjustment-data-bridge-configmap.yaml
-  - data-bridge/cse-valid-idcc-ttc-adjustment-data-bridge-deployment.yaml
-  - data-bridge/cse-valid-idcc-ttc-adjustment-data-bridge-service.yaml
+  - data-bridge/cse-valid-idcc-data-bridge-configmap.yaml
```

Comment:
Why did you not move the files to the parent folder, as you did for other processes?